db_scripts/migrations/R__03_available_columns.sql

```diff
@@ -0,0 +1,194 @@
+INSERT INTO available_columns (
+    column_name,
+    table_name,
+    group_name,
+    readable_column_name
+) VALUES
+('address', 'members', 'Datos Personales', 'Direccion'),
```

Comment:
```suggestion
('address', 'members', 'Datos Personales', 'Dirección'),
```

Creo que sería bueno tomar en cuenta la ortografía en todo este archivo, porque sino sería una nueva migración para corregir estas cosas